### PR TITLE
Debugging Finds

### PR DESCRIPTION
**Pull request by TheBurchLog**
I was testing out v3 and discovered that we don't cache the current gardens at startup. So I updated that and also updated how we handle garden update events to cache the results.

Discovered that we were mapping the wrong field from the config for default namespace for plugins. Fixed that mapping

### CODE REVIEW

File: src/app/example_configs/config.yaml [hunk 1]
```diff
@@ -104,7 +104,7 @@ event:
         public_key: null
       url_prefix: /
 garden:
-  name: default
+  name: parent_garden
```
Comment (hazmat345) [inline src/app/example_configs/config.yaml]: This shouldn't be updated in version control

File: src/app/example_configs/config.yaml [hunk 2]
```diff
@@ -116,7 +116,7 @@ metrics:
     port: 2339
     url: http://localhost:3000
 namespaces:
-  local: default
+  local: parent
```
Comment (hazmat345) [inline src/app/example_configs/config.yaml]: This shouldn't be updated in version control

File: src/app/beer_garden/systems.py [hunk 3]
```diff
@@ -72,7 +72,7 @@ def create_system(system: System) -> System:
             system.max_instances = len(system.instances)
 
     if system.namespace is None:
-        system.namespace = beer_garden.config.get("namespaces.name")
+        system.namespace = beer_garden.config.get("namespaces.local")
```
Comment (hazmat345) [inline src/app/beer_garden/systems.py]: Should this be using the garden name instead? I think we should get rid of that whole namespaces config section as it doesn't really make sense anymore.

**Comment by hazmat345** [comment]
@TheBurchLog I added a couple things, but I'm going to go ahead and merge this.

- I reverted the changes to the config file
- The entry points need to know about the gardens from the database as well.

File: src/app/example_configs/config.yaml (hunk 2 above)
Comment (TheBurchLog) [reply src/app/example_configs/config.yaml]: yeah, that should not have made it in

**Pull request merged**